src/main/scripts/releases/3.2/insertParatooProtocolConfig.js

```diff
@@ -0,0 +1,451 @@
+var protocols = {
+    "068d17e8-e042-ae42-1e42-cff4006e64b0":
+        {
+            "name": "Opportune",
+            "usesPlotLayout": false,
+            "tags": ["survey"],
+            "apiEndpoint": "opportunistic-surveys",
+            "overrides": {
+                "dataModel": {
+                    "opportunistic-observation.number_of_individuals": {
+                        "dwcAttribute": "individualCount"
+                    }
+                },
+                "viewModel": null
+            }
+        },
+    "cd2cbbc7-2f17-4b0f-91b4-06f46e9c90f2": {
+        "name": "Vegetation Mapping",
+        "usesPlotLayout": false,
+        "tags": ["survey"],
+        "apiEndpoint": "vegetation-mapping-surveys",
+        "geometryType": "Point",
+        "geometryPath": "attributes.position",
+        "startDatePath": "attributes.vegetation_mapping_survey.data.attributes.start_date_time",
+        "endDatePath": null,
+        "surveyIdPath": "attributes.vegetation_mapping_survey.data.attributes.surveyId"
+    },
+    "a9cb9e38-690f-41c9-8151-06108caf539d":
+        {
+            "name": "Plot Selection",
+            "usesPlotLayout": false,
+            "apiEndpoint": "plot-selections",
+            "tags": ["site"]
+        },
+    "d7179862-1be3-49fc-8ec9-2e219c6f3854": {
+        "name": "Plot Layout and Visit",
+        "usesPlotLayout": false,
+        "apiEndpoint": "plot-definition-surveys",
+        "tags": ["site"]
+    },
+    "617df00c-0e4f-4267-9efc-9ca9eae19686": {
+        "name": "Plot Description(enhanced)",
+        "apiEndpoint": "plot-description-enhanceds",
+        "usesPlotLayout": true,
+        "tags": ["site"]
+    },
+    "dc10f902-e310-45eb-b82a-bebab050b46b": {
+        "name": "Plot Description(standard)",
+        "apiEndpoint": "plot-description-standards",
+        "usesPlotLayout": true,
+        "tags": ["site"]
+    },
+    "3cbc5277-45fb-4e7a-8f33-19d9bff4cd78": {
+        "name": "Drone Survey", "usesPlotLayout": false, "tags": ["survey"],
+        "apiEndpoint": "drone-surveys"
+    },
+    "3d2eaa76-a610-4575-ac30-abf40e57b68a": {
+        "name": "Dev sandbox", "usesPlotLayout": false, "tags": ["development"],
+        "apiEndpoint": "dev-sandbox-surveys"
+    },
+    "5fd206b5-25cb-4371-bd90-7b2e8801ea25": {
+        "name": "Photopoints - DSLR Panorama",
+        "usesPlotLayout": true,
+        "tags": ["survey"],
+        "apiEndpoint": "photopoints-surveys",
+    },
+    "383fa013-c52d-4186-911b-35e9b2375653": {
+        "name": "Photopoints - Compact Panorama",
+        "usesPlotLayout": true,
+        "tags": ["survey"],
+        "apiEndpoint": "photopoints-surveys",
+        "startDatePath": "attributes.start_date_time",
```

Comment:
The "attributes" prefix is no longer valid - I should have edited this when I made my change to get the dates and site from the /reverse-lookup call.

Reply:
fixed